### DISCUSSION TITLE
Qs.

### DISCUSSION THREAD

**Discussion by cloudcompute** [bravecobra/k8s-dev-infrastructure#593] [Q&A] [answered]
Hi @bravecobra

Is there any specific reason for using Traefik and not nginx ingress? In a local development scenario, does Traefik serve some sepcific purpose?

Can I use a tool like [skaffold](https://github.com/GoogleContainerTools/skaffold) with this project?

Thanks

**Comment by bravecobra** [comment, marked as answer]
No specific reason. The idea was/is to expand and support other reverse proxies too.

No, skaffold cannot be used directly, yerraform handles that way better in my opinion. However you can learn from the helm configs used here and use them in your skaffold code. If you look at the history you see I used skaffold before but replaced it with terraform later on as it better fitted for this project.
